content/doc/book/pipeline/using-credentials.adoc

```diff
@@ -32,7 +32,7 @@ pages is provided by the plugin:credentials-binding[Credentials Binding plugin].
 
 .The correct way to handle credentials in Jenkins
 video::yfjtMIDgmfs[youtube,width=800,height=420]
-
+You can learn how to manage and administer credentials in jenkins through the above linked youtube video.
```

Comment:
The original phrasing of https://github.com/jenkins-infra/jenkins.io/issues/5490 was incomplete.  The information in the "Using credentials" page of "Using Jenkins" also needs to be included in the "Pipeline" section, or needs to be linked there.

If we move the page from one location to another, we break the links that pointed to the old page and we lose the link to that page in the "Using Jenkins" section.

Rather than moving this file from one folder to another, it would be better to add a link to the existing page into the one or more of the "Using Pipelines" pages.

When I look at this, I think that we already have it implemented.  The ["Using a Jenkinsfile" page](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/) includes a section ["Handling credentials"](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#handling-credentials).  That page would benefit by having the video embedded in the page immediately after the link to the "Using credentials" page.

```suggestion

```

Since this pull request includes a move of the file, you can either undo the move in a new commit or you could start a new pull request and close this pull request.  If you need help with any of those git operations, you're welcome to join us in [Docs Office Hours](https://www.jenkins.io/sigs/docs/#office-hours).

Reply:
Okay sir, I will make the changes and let you know with a new pull request.
Also would like to inform that Docs Office Hours link that you have given is not working. 
Thanks